.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules/
 out/
 third_party/mongodb/
 third_party/node/
+.babel-cache.json
```

Comment:
Just question: Why do we need this? In 2.0, we are just using TypeScript instead of Babel.
If some codes are using babel, we should get rid of them.

Reply:
You are right. It was a unnecessary file which generated by 1.0 ver.
I will update it.